modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -2242,25 +2266,57 @@ private void _testGetUserAccountWithNestedFields() throws Exception {
 			String.valueOf(user.getUserId()), role.getRoleId(),
 			new String[] {ActionKeys.DELETE});
 
+		AssetEntry assetEntry = _assetEntryLocalService.fetchEntry(
+			_classNameLocalService.getClassNameId(User.class),
+			user.getUserId());
+
+		AssetVocabulary assetVocabulary = AssetTestUtil.addVocabulary(
+			TestPropsValues.getGroupId());
+
+		AssetCategory assetCategory = AssetTestUtil.addCategory(
+			TestPropsValues.getGroupId(), assetVocabulary.getVocabularyId());
+
+		_assetEntryAssetCategoryRelLocalService.addAssetEntryAssetCategoryRel(
+			assetEntry.getEntryId(), assetCategory.getCategoryId());
+
```

Comment:
I don't know if they should go here or they should go first for be alphabetically ordered. Where we put the same in the AccountResourceTest?
The only reason to stay here is that are the data for the `taxonomyCategoryBriefs` that is required as last in the nestedFields parameter